modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -487,4 +496,43 @@ public static void closeAll(Collection<? extends AutoCloseable> closeables) thro
     public static void closeAll(AutoCloseable... closeables) throws Exception {
         closeAll(Arrays.asList(closeables));
     }
+
+    /**
+     * Short date format pattern for log messages in "quiet" mode.
+     * Only time is included since we don't expect "quiet" mode to be used
+     * for longer runs.
+     */
+    private static final SimpleDateFormat SHORT_DATE_FMT = new SimpleDateFormat("HH:mm:ss");
```

Comment:
First, why not declare it as DateFormat instead of specific class? Usually field type is more general than type of the instance. I notice such pattern regularly in your code.
Second, SimpleDateFormatter is not threadsafe, you should probably use some DateTimeFormatter instead

Reply:
1) I really like to avoid abstraction where is not need, but here is a copy-paste.
2) Yes, you right, but this issue never worried us before in Ignite 2.

I think this method is convenient to use in debug (possible in tests and seldom in product's code) by this reason I added it here. But it just a copy from Ignite 2.